### PR TITLE
New paragraph about saving snippets to disk

### LINKED ISSUE #913: Where are snippets saved?
Please provide documentation on where the snippets are saved.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 71876e38-721a-4384-4c79-ca3a196d1384
* Version Independent ID: 20e75670-1789-5f4e-c361-2d07dfdd68e0
* Content: [Sources Panel Overview - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/sources)
* Content Source: [microsoft-edge/devtools-guide-chromium/sources.md](https://github.com/MicrosoftDocs/edge-developer/blob/master/microsoft-edge/devtools-guide-chromium/sources.md)
* Product: **microsoft-edge**
* Technology: **devtools**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by captainbrosset**
This fixes #913 by adding some information about how to save snippets to disk.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/javascript/snippets.md [hunk 1]
```diff
@@ -153,7 +153,15 @@ To rename your new Snippet with a custom name, navigate to [Rename Snippets](#re
...
 1.  [Open the Snippets tab](#open-the-snippets-tab).  
 1.  Hover on the Snippet name, open the contextual menu \(right-click\), and choose **Remove**.  
-    
+
+## Save Snippets  
+
+By default, Snippets are only available within DevTools, but you can also save them to disk.  
+
+1.  [Open the Snippets tab](#open-the-snippets-tab).  
+1.  Hover on the Snippet name, open the contextual menu \(right-click\), and choose **Save as...**.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/javascript/snippets.md:162]:
```suggestion
1.  Hover on the Snippet name, open the contextual menu \(right-click\), and select **Save as**.  
```
You might want to Edit this Suggestion to continue to use 'choose' per rest of article.  But, "..." should be deleted, per Microsoft Writing Style Guide.  Public version of "Ellipses" page https://docs.microsoft.com/en-us/style-guide/punctuation/ellipses says "When there's an ellipsis in UI, don’t include it in instructions or procedures.  Example: Select Safety, and then Delete browsing history."

File: microsoft-edge/devtools-guide-chromium/javascript/snippets.md [hunk 2]
```diff
@@ -153,7 +153,15 @@ To rename your new Snippet with a custom name, navigate to [Rename Snippets](#re
 
 1.  [Open the Snippets tab](#open-the-snippets-tab).  
 1.  Hover on the Snippet name, open the contextual menu \(right-click\), and choose **Remove**.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/javascript/snippets.md:155]: If you want to do a word-change throughout the entire article as part of this PR: change some instances of 'choose' to 'select', per Microsoft Writing Style Guide.  Public version of important page "Describing interactions with UI": https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Minimum change: remove ellipses.
Bigger, optional change: throughout article, apply correct usage of 'select' instead of 'choose', per Microsoft Writing Style Guide.  'select' is almost always correct, instead of 'choose', per MWSG.

**Comment by captainbrosset** [comment]
Thank you @mikehoffms. I did the change throughout the article to change almost all occurrences of `choose` with `select`. I left `choose` in 3 sentences since they were about choosing a file or snippets name (so, based on the customer's preference).
I also removed the ellipsis.

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**